e2e/run.py

```
@@ -0,0 +1,211 @@
+
+    def _fill(self, text: str) -> str:
+        return (text.replace("{agent_a}", self.cfg.agent_a)
+                    .replace("{agent_b}", self.cfg.agent_b))
+
+    # -- scenario ----------------------------------------------------------
+    def run_scenario(self, sc: dict, phase_after_232: bool) -> Result:
+        out: list[str] = []
+        if sc.get("playwright"):
+            out.append("  playwright scenario — run steps manually / via playwright lane:")
+            out += [f"    - {s}" for s in sc.get("steps", [])]
+            return Result(sc["id"], sc["title"], True, out + ["  SKIPPED (playwright lane)"])
```

Ruff (0.16.2)

[warning] Consider `[*out, "  SKIPPED (playwright lane)"]` instead of concatenation

Replace with `[*out, "  SKIPPED (playwright lane)"]`

(RUF005)